README.md

```diff
@@ -1,9 +1,11 @@
-# Introduction
+# Bill of Materials for Jenkins plugins
```

Comment:
the confusion is because this is not "a bill of materials" as the dependencies have never been shipped together.

to me the repo should be renamed.

Reply:
I kinda agree, can most likey be renamed `plugin-bom` to have the same naming as `plugin-pom`

Reply:
I would not mind, needs sign-off from @jglick and @timja

Reply:
Do not much care about the _repository_ but renaming the _artifact_ would be intrusive.